<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsSimulatorBasicsComponent</class>
 <widget class="QFrame" name="CSettingsSimulatorBasicsComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulator basic settings</string>
  </property>
  <layout class="QGridLayout" name="gl_SettingsSimulatorBasics">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="2" column="1" rowspan="4">
    <widget class="QPlainTextEdit" name="pte_ModelDirectories">
     <property name="placeholderText">
      <string>Model directories</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_Simulator">
     <property name="text">
      <string>Simulator:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pb_AdjustModelDirectory">
     <property name="toolTip">
      <string>remove redundant directories, fix file paths ...</string>
     </property>
     <property name="text">
      <string>adjust</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbl_ExcludeDirectories">
     <property name="toolTip">
      <string>Excluded from model loading</string>
     </property>
     <property name="text">
      <string>Exclude directory patterns:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" alignment="Qt::AlignLeft">
    <widget class="swift::gui::components::CSimulatorSelector" name="comp_SimulatorSelector">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="hs_Buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pb_Save">
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pb_Reset">
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_SimulatorDirectory">
     <property name="text">
      <string>Simulator directory:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_ModelDirectory">
     <property name="text">
      <string>Model directories:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pb_SimulatorFileDialog">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pb_ExcludeFileDialog">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" rowspan="4">
    <widget class="QPlainTextEdit" name="pte_ExcludeDirectories">
     <property name="documentTitle">
      <string>Excluded directory patterns</string>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="placeholderText">
      <string>Excluded directory patterns</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_SimulatorDirectory">
     <property name="placeholderText">
      <string>Simulator directory path</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="pb_CopyDefaults">
     <property name="toolTip">
      <string>copy (materialize) defaults</string>
     </property>
     <property name="text">
      <string>copy def.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pb_ModelFileDialog">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="lbl_ModelDirsInfo">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/diagona/icons/diagona/icons/exclamation--frame.png&quot;/&gt; Changing model directories means you have to update your model set! Check documentation on &amp;quot;creating a model set&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_SimulatorDirectory</tabstop>
  <tabstop>pte_ModelDirectories</tabstop>
  <tabstop>pte_ExcludeDirectories</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
